src/components/ChangePlan/Card/Card.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+
+const Card = ({ data }) => {
```

Comment:
What do you think about deconstructing data here to make it clear what data do you accept?